### ISSUE TITLE
es connector failed with presto

### ISSUE THREAD

**Issue by ccz414172041** [prestodb/presto#13345]
I want connect es with presto ,but I execute query "select * from es_catalog.schema.tableName" 

Query failed (#20190906_072117_00000_vzwbk): NoNodeAvailableException[None of the configured nodes are available: [{#transport#-1}{q2HDbxS4TbSTS0l1X5zfLw}{10.10.17.159}{10.10.17.159:9200}]]

my es.properties:

connector.name=elasticsearch
elasticsearch.scroll-size=1000
elasticsearch.request-timeout=30s
elasticsearch.max-request-retries=30
elasticsearch.max-request-retry-time=30s

my es.json:

{
  "tableName": "doctest",
  "schemaName": "es_test",
  "host": "10.10.17.159",
  "port": "9200",
  "clusterName": "my-application",
  "index": "bank",
  "type": "account",
  "columns": [

      {
          "name": "address",
          "type": "varchar",
          "jsonPath": "address",
          "jsonType": "varchar",
          "ordinalPosition": "0"
      }
  ]
}

this is my es server properties:
{
    "name": "Aqueduct",
    "cluster_name": "my-application",
    "version": {
        "number": "2.3.5",
        "build_hash": "90f439ff60a3c0f497f91663701e64ccd01edbb4",
        "build_timestamp": "2016-07-27T10:36:52Z",
        "build_snapshot": false,
        "lucene_version": "5.5.0"
    },
    "tagline": "You Know, for Search"
}

my port is : localhost:9200/

**Comment by vkorukanti** [comment]
@ccz414172041 The current connector doesn't support HTTP transport. Change the port `9200` to native transport port `9300` (default). After https://github.com/prestosql/presto/pull/1213, it should support both native and http transport.

**Comment by ccz414172041** [comment]
thank you for you answer!
but I tried change the port to 9300, there are still the same error that NoNodeAvailableException[None of the configured nodes are available: [{#transport#-1}{q2HDbxS4TbSTS0l1X5zfLw}{10.10.17.159}{10.10.17.159:9300}]].
I never change the elasticsearch.yml, I saw some issues of the same problem. The answer is check my config of the cluster_name or port, but I confirm my config is correct.
 Maybe my version is too low?
@vkorukanti
